ui/raidboss/data/06-ew/raid/p12s.ts

```diff
@@ -447,6 +448,18 @@ const triggerSet: TriggerSet<Data> = {
       },
       default: 'shapeAndDebuff',
     },
+    {
+      id: 'classicalConcepts2ActualNoFlip',
+      comment: {
+        en:
+          'Suppresses initial output & calls final position immediately in chosen pair order (e.g. no flip).  <a href="https://user-images.githubusercontent.com/67395342/258580930-1659573f-a492-4839-a275-e51449bb85c0.gif" target="_blank">Visual</a>',
```

Comment:
```suggestion
          'Only calls final position immediately in chosen pair order with no flip.  For example, for BPOG, the crosses will be far west. <a href="https://user-images.githubusercontent.com/67395342/258580930-1659573f-a492-4839-a275-e51449bb85c0.gif" target="_blank">Visual</a>',
```

Also, can you check this image in somewhere so that we're not depending on github hosting some image from a comment and instead we can link to it in github pages?

Reply:
> Also, can you check this image in somewhere so that we're not depending on github hosting some image from a comment and instead we can link to it in github pages?

Sure, makes sense.  When you say check it in somewhere, do you mean like external image hosting svc (e.g. imgur), or do you actually want it checked in to this repo? (e.g. new subdir under resources maybe?)

Reply:
Yeah, I mean in the repo itself so it doesn't get lost.

And yeah, I wasn't sure where a good place was. It's not really (but sort of?) `docs/`, but I think `resources/images/` or something along those lines would be good.

Reply:
> Yeah, I mean in the repo itself so it doesn't get lost.
> 
> And yeah, I wasn't sure where a good place was. It's not really (but sort of?) `docs/`, but I think `resources/images/` or something along those lines would be good.

I'm having trouble getting the animated gif file added to the repo.  It looks fine on my local fs, but when I commit it to the feature branch and sync, the file gets added to github without any animation (and with the bottom third of the image blank).  I just created a test feature branch and had the same thing happen.  Tried committing with both VS2022 and git bash (in case it was a weird VS bug).  And when I redownload the gif from the github branch, the downloaded file has the same display issue (and, incidentally, is 237 bytes smaller than the original source file).

I'm at a bit of a loss.  Any ideas?  Maybe some sort of line-ending issue since it's being uploaded from a windows machine?

Reply:
Mmm is it being added as text instead of binary? If you add `*.gif binary` to `.gitattributes` and then add the file, does it work ok?

Reply:
> Mmm is it being added as text instead of binary? If you add `*.gif binary` to `.gitattributes` and then add the file, does it work ok?

Yep, looks like that did the trick.  Thanks!